ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/MiniProtocol/ChainSync/Client.hs

```diff
@@ -522,8 +522,8 @@ chainSyncClient mkPipelineDecision0 tracer cfg
         (theirFrag, theirHeaderStateHistory) <- do
           case attemptRollback intersection (ourFrag, ourHeaderStateHistory) of
             Just (c, d) -> return (c, d)
-            -- The @intersection@ is not on the candidate chain, even though
-            -- we sent only points from the candidate chain to find an
+            -- The @intersection@ is not on our current chain, even though
+            -- we sent only points from our current chain to find an
```

Comment:
Same point about "our current chain" -> "our fragment" as above.

Reply:
Adopted in a38ce67.